backend/sync.js

```diff
@@ -249,7 +249,7 @@ function makeBloomFilter(backend, lastSync) {
 function getChangesToSend(backend, have, need) {
   const opSet = backend.get('opSet')
   if (have.length === 0) {
-    return need.map(hash => OpSet.getChangeByHash(opSet, hash))
+    return need.map(hash => OpSet.getChangeByHash(opSet, hash)).filter(change => change)
```

Comment:
this construct was a little tough to read. I might say
```
.filter(change => change != null)
```

Reply:
Ok. a5c8bbd4fa94f476f781d2318a54b5154dc02800